code/game/machinery/dna_infuser/organ_sets/goliath_organs.dm

```diff
@@ -113,29 +113,44 @@
 	toolspeed = 0.1
 	/// Amount of damage we deal to the mining and boss factions.
 	var/mining_bonus_force = 80
+	COOLDOWN_DECLARE(tendril_hammer_cd)
```

Comment:
You should doc comment these the same way as var declarations (given that it is one. with a weird wrapper)

Reply:
Sure